src/Services/Import.php

```diff
@@ -26,7 +28,8 @@ public function __construct(string $type, UploadedFile $file, array $params = []
 
     public function handle(): self
     {
-        $this->init()
+        $this->checkAlreadyRunning()
```

Comment:
we don't want that, sometimes we actually want to import more than one file of the same type, at the same time.